packages/scss/src/components/_button.scss

```diff
@@ -153,11 +155,28 @@
 // STATES
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
+$buttonDisabled: .5;
```

Comment:
Right. As I think this opacity could be used in multiple components, I set it in "commons.disabled.opacity".